controllers/managedocs_controller_test.go

```diff
@@ -46,14 +45,67 @@ var _ = Describe("ManagedOCS controller", func() {
 	}
 
 	Context("reconcile()", func() {
-		When("there is no storage cluster resource in the cluster", func() {
-			It("should create a new storage cluster", func() {
+		When("There is no addonParam secret in the cluster", func() {
```

Comment:
Test descriptions should be human-readable, as such let's change 'addonParam secret"  to "add-on parameters secret"

Reply:
Done